### PR TITLE
SW-2348: Responsive Page Layouts -> Empty -> Inventory

### PR DESCRIPTION
**Pull request by joshbuchea**
This PR includes changes to align several pages with the [empty state designs in figma](https://www.figma.com/file/aLIG1C6uIMyBWiTopSluxe/Visual-Refresh?node-id=853%3A112604&t=FSN08oFo2ACctbR6-0):

- Accessions
- Monitoring
- Inventory

## Notes

-  The background for the `NavBar` should be transparent so the background image is visible behind. @charlie-tf is currently making some changes re: to background transparency and this will be resolved in another PR soon.
- There are still blue hyperlinks in some components, this will be resolved in another PR soon

## Screenshots

Accessions:

localhost_3000_inventory (3)

Monitoring:

localhost_3000_planting-sites (1)

Inventory:

localhost_3000_planting-sites

### CODE REVIEW

File: src/components/seeds/database/index.tsx [hunk 1]
```diff
@@ -565,7 +565,7 @@ export default function Database(props: DatabaseProps): JSX.Element {
           />
         </>
       )}
-      <TfMain>
+      <TfMain backgroundImageVisible={!isOnboarded}>
```
Comment (karthikbtf) [inline src/components/seeds/database/index.tsx:568]: @joshbuchea the screenshots in the PR seem to indicate otherwise. Do they need to be updated?

**Review by karthikbtf — APPROVED** [review]
Changes look good 🙌 
One comment re: screenshots

File: src/components/seeds/database/index.tsx (hunk 1 above)
Comment (joshbuchea) [reply src/components/seeds/database/index.tsx:568]: @karthikbtf I _think_ the screenshots accurately reflect the code change 🤔  The background image should be visible/shown, when the user hasn't completed onboarding...

Comment (joshbuchea) [reply src/components/seeds/database/index.tsx:568]: Oh unless you're referring to the background for the `NavBar`. In that case, there's a note in the PR description.

Comment (karthikbtf) [reply src/components/seeds/database/index.tsx:568]: ugh yes my bad, was looking at the navbar transparency - totally unrelated

Comment (karthikbtf) [reply src/components/seeds/database/index.tsx:568]: sorta related but unrelated to this PR

Comment (joshbuchea) [reply src/components/seeds/database/index.tsx:568]: It's all good, it's related. I appreciate your help with thorough PR reviews & testing. Thanks! 🙌

**Pull request merged**